frontend/sentry.client.config.ts

```diff
@@ -21,7 +21,5 @@ Sentry.init({
          'next.runtime': 'client',
       },
    },
-   ignoreErrors: [
-      'TypeError: NetworkError when attempting to fetch resource.'
-   ],
+   ignoreErrors: ['TypeError: NetworkError when attempting to fetch resource.'],
```

Comment:
Was this linter action supposed to be triggered?

Reply:
Basically, I see that the action just runs prettier and if there were changes it commits those changes. However, since your pull did not modify `frontend/sentry.client.config.ts` I am assuming this file was pushed to `main` without it being formatted first and is now being formatted with the GH linter?

Reply:
Correct, this file was pushed to main without being formatted. The action formatted it on this branch and pushed it in a new commit.

Going forward, we shouldn't encounter this situation because feature branches will be formatted by the action before being merged into main.